bigchaindb/log.py

```diff
@@ -1,8 +1,14 @@
+import bigchaindb
+import functools
 import logging
+
+from bigchaindb.common.exceptions import ConfigurationError
+from logging.config import dictConfig as set_logging_config
 from os.path import expanduser, join
 
 
 DEFAULT_LOG_DIR = expanduser('~')
+BENCHMARK_LOG_LEVEL = 60
```

Comment:
After discussion with @shahbazn it seems that having `BENCHMARK_LOG_LEVEL=15` would be more suitable as the benchmark logs would not show up when a production system is started with levels, `INFO`, `WARNING`, `ERROR` or `CRITICAL`.